org.alloytools.util/src/main/java/org/allotools/conversion/util/DTOs.java

```
@@ -0,0 +1,114 @@
+package org.allotools.conversion.util;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Type;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import aQute.lib.converter.Converter;
+
+public class DTOs {
+
+    @SuppressWarnings("unchecked" )
+    public static <T> T copy(T from, T to) {
+
+        Class< ? extends Object> class1 = from.getClass();
+        assert class1 == to.getClass();
+
+        if (to == null) {
+            try {
+                to = (T) class1.newInstance();
+            } catch (Exception e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            }
+        }
+
+        for (java.lang.reflect.Field toField : to.getClass().getFields()) {
+            try {
+                java.lang.reflect.Field fromField = class1.getField(toField.getName());
+                Object fromValue = fromField.get(from);
+                Object toValue = convert(toField.getGenericType(), fromValue);
+                toField.set(to, toValue);
+            } catch (Exception e) {
+                //
```

what about this?  Is it ok to just ignore this exception?  If so, add a comment